bitnami/redis/Chart.yaml

```diff
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/redis
-version: 17.0.8
\ No newline at end of file
+version: 18.0.0
```

Comment:
I don't think we should bump a major version because of the change in `sentinel.failoverTimeout`. 
The change is not strictly non-backward compatible and increasing a timeout should not break anything. 

Could you please bump the version `17.0.11`? (latest at this moment is 17.0.10)

Reply:
Sounds good. Updated it.